### ISSUE TITLE
SolO IRFU quicklooks: "Index exceeds the number of array elements (3).", crashes

### ISSUE THREAD

**Issue by ErikPGJ** [irfu/irfu-matlab#98]
### Step 1: Latest code?
- [X] This issue is present in the most recent code (i.e. after running ```git pull``` this is still a problem).
  Problem found in 7a6b2352.

### Step 2: Describe your environment
 * irfu-matlab branch (e.g. ```master```, ```devel```): devel
 * Matlab version used (e.g. ```Matlab R2020b```): R2020b
 * Operating system (e.g. ```Windows 10```, ```Mac OS X 10.15.6```, ```Linux Ubuntu 20.04```): Ubuntu 18.04 (brain)
 - [X] A 64-bit installation of Matlab was used.

#### Expected behavior

Not crash. Produce weekly plots.

#### Actual behavior

Crashed when executed command on brain:
`solo.quicklook_main('/home/erjo/so_irfu-matlab_qli//mission/solar_orbiter/+solo/irf_logo.png', '/data/solo/data_yuri/', '/home/erjo/temp/qli/so_qli_MTEST.2022-08-09_09.51.07._2021-08-20T00:00:00---2021-09-10T00:00:00', '1', '1', '2021-08-20T00:00:00', '2021-09-10T00:00:00');
`

```
  [warning: get_variable(60)] dataobj:get_variable:cyclic_depCyclic dependecy: Epoch <-> Half_interval
  [warning: get_variable(60)] dataobj:get_variable:cyclic_depCyclic dependecy: Epoch <-> Half_interval
  [warning: get_variable(60)] dataobj:get_variable:cyclic_depCyclic dependecy: Epoch <-> Half_interval
exception.message=Index exceeds the number of array elements (3).
quicklooks_7days, row 102
quicklook_main, row 304
Command exited with non-zero status 99
```

Last completed plot is `1w/20210825T00_20210901T00.png` so presumably the problem is triggered when enabling weekly plots for 2021-09-01T00 to 2021-09-08T00.

**Comment by ErikPGJ** [comment]
Get same error message, presumably same bug, for other time period:
`
solo.quicklook_main('/home/erjo/so_irfu-matlab_qli//mission/solar_orbiter/+solo/irf_logo.png', '/data/solo/data_yuri/', '/home/erjo/temp/qli/so_qli_MTEST.2022-08-10_10.18.00._2020-09-10T00:00:00---2020-10-01T00:00:00', '1', '1', '2020-09-10T00:00:00', '2020-10-01T00:00:00');`

No weekly plots produced.

**Comment by JordiBoldu** [comment]
Bug originated in a for loop to increase the date to open new PAS and TNR data. for iii = 1:(Tint(2)-Tint(1))/3600/24. If the input Tint has days without files it will pop out an error. 
Changed previously for PAS data, but forgot to do it for TNR.

Changed to for iii = 1:ceil((myFile(end).stop-myFile(1).start)/3600/24), so it will iterate only the amount of times equal to the number of files.

**Comment by ErikPGJ** [comment]
Appears to have been fixed with fbc3523e.
